Library/RefTag.swift

```diff
@@ -175,38 +159,7 @@ public enum RefTag {
   }
 }
 
-extension RefTag: Equatable {
-}
-public func == (lhs: RefTag, rhs: RefTag) -> Bool {
```

Comment:
Oh, Swift 4.2 auto-synthesizes `Equatable` conformance where the types involved also conform to `Equatable`. I just removed it here because it could auto-synthesize. We should probably add an investment day task to do this more widely in the codebase.

Just below these lines that were removed you'll see:

```swift
extension RefTag: Equatable {}
```